diff --git a/lib.php b/lib.php
--- a/lib.php
+++ b/lib.php
@@ -245,11 +245,10 @@ function certificate_supports($feature) {
     switch ($feature) {
 
         case FEATURE_GROUPS:                  return true;
         case FEATURE_GROUPINGS:               return true;
-        case FEATURE_GROUPMEMBERSONLY:        return true;
         case FEATURE_MOD_INTRO:               return true;
+        case FEATURE_SHOW_DESCRIPTION:        return true;
         case FEATURE_COMPLETION_TRACKS_VIEWS: return true;
         case FEATURE_BACKUP_MOODLE2:          return true;
         default: return null;
     }
 }
diff --git a/version.php b/version.php
--- a/version.php
+++ b/version.php
@@ -19,7 +19,7 @@
  *
  * @package    mod_certificate
  * @copyright  Mark Nelson <markn@moodle.com>
- * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or late
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
 defined('MOODLE_INTERNAL') || die();
@@ -27,10 +27,10 @@ defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2020082500; // The current module version (Date: YYYYMMDDXX)
-$plugin->requires  = 2016052300; // Requires this Moodle version (3.1)
+$plugin->version   = 2025100100; // The current module version (Date: YYYYMMDDXX)
+$plugin->requires  = 2022112800; // Requires Moodle 4.1 (2022112800)
 $plugin->cron      = 0; // Period for cron to check this module (secs)
 $plugin->component = 'mod_certificate';
 
 $plugin->maturity  = MATURITY_STABLE;
-$plugin->release   = "Master (Build: 2020082500)"; // User-friendly version number
+$plugin->release   = "Moodle 4.1 compatibility update (Build: 2025100100)"; // User-friendly version number